### PR TITLE
Add network name to data.json

### PR DESCRIPTION
**Pull request by ishleenk17**
- Bug

## What does this PR do?
This PR adds network names to the data example of host datastream

## Checklist

- [x] My code follows the style guidelines of this project
- [] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

- Closes https://github.com/elastic/beats/issues/32461

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -78,8 +78,15 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 
 *Metricbeat*
 
+
```
Comment (lalit-satapathy) [inline CHANGELOG.next.asciidoc:81]: Extra line not needed.

File: CHANGELOG.next.asciidoc [hunk 2]
```diff
@@ -78,8 +78,15 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
...
+- update kubernetes apiserver metricset to not collect deprecated metrics and fix dashboard {pull}31973[31973]
+- Check for nil metadata in GCP. {pull}32281[32281]
+- update kubernetes controllermanager metricset to not collect deprecated metrics and fix dashboard {pull}32037[32037]
+- Fix to ARN parsing for Cloudwatch resource names with leading slashes {pull}32358[32358]
 - in module/windows/perfmon, changed collection method of the second counter value required to create a displayable value {pull}32305[32305]
 - Fix and improve AWS metric period calculation to avoid zero-length intervals {pull}32724[32724]
+- Add network name to data.json {pull}32981[32981]
+
```
Comment (lalit-satapathy) [inline CHANGELOG.next.asciidoc:89]: Extra line not needed.

File: CHANGELOG.next.asciidoc [hunk 3]
```diff
@@ -78,8 +78,15 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 
 *Metricbeat*
 
+
+- update kubernetes apiserver metricset to not collect deprecated metrics and fix dashboard {pull}31973[31973]
```
Comment (lalit-satapathy) [inline CHANGELOG.next.asciidoc:82]: Can you please check why these un-related change logs showing up in addition here?

**Comment by ishleenk17** [comment]
Some issues with this PR. 
Updated to https://github.com/elastic/beats/pull/32986/

**Pull request closed without merging**